### ISSUE TITLE
`text-muted` text color does not pass WCAG 2.0 AAA requirements 

### ISSUE THREAD

**Issue by trackleft** [az-digital/arizona-bootstrap#166]
## User Story(s)

As a _Visitor_, I would like to be able to read all of the text on the page. In order to do so, the disabled and date colors should bump up in contrast to match WCAG AAA guidelines for accessibility in regards to color contrast.

Some of the content that is displayed with low contrast colors are arguably important to the context of the content on the page.

See : https://wave.webaim.org/report#/https://digital.arizona.edu/arizona-bootstrap/docs/2.0/utilities/colors/

## Conditions of satisfaction
- [ ] Should pass color contrast testing from Dubbot, Lighthouse and Webaim
- [ ] New `text-muted` color should be set to `#51595C`  everywhere.

### Roles

* _Visitor_ of an Arizona Quickstart website
* _Member_ of the Arizona Digital team
* _Admin_ of the Arizona Digital team

### A good user story should be (I-N-V-E-S-T principle)

* _Independent_ (from other user stories, allowing to realize them in any order)
* _Negotiable_ (omit details that would freeze the story)
* _Valuable_ (implementation delivers an increment of functionality, observable by and useful to users)
* _Estimatable_ (developers should be able to estimate its size relative to other stories)
* _Sizable_ (implementation fits in one iteration – if it needs many to complete, it is an EPIC)
* _Testable_ (user must be able to check the conditions of satisfaction). -->

**Comment by joeparsons** [comment]
Would an alternate fix be to override the value of the Bootstrap `$text-muted` variable to be `$gray-700` instead of `$gray-600`?

**Comment by smayden** [comment]
Looking at this ticket, I'm wondering if this work has already been done and merged? 

This PR addressed the text-muted color and was merged in Nov. 2020 https://github.com/az-digital/arizona-bootstrap/pull/167

Tested text with the text-muted class in QS2 and found no accessibility problems with it in Lighthouse. Color is set to #495057 which matches the gray-700 color suggested on this ticket in October.

Am I missing something?

**Comment by danahertzberg** [comment]
Agreed. It looks like this ticket can be closed with the merged PR #167
